data/Hong_Kong/Legislative_Council/sources/instructions.json

```diff
@@ -1,14 +1,9 @@
 {
   "sources": [
     {
-      "file": "morph/data.csv",
-      "create": {
-        "from": "morph",
-        "scraper": "struan/hong_kong_legislative_council_members",
-        "query": "SELECT *, REPLACE(id,'yr12-16/','') AS id FROM data ORDER BY id"
-      },
+      "file": "manual/term-5.csv",
       "source": "http://www.legco.gov.hk/",
-      "type": "membership"
+      "type": "membership",
```

Comment:
This isn't valid JSON, so makes a build fail with

```
JSON::ParserError: parse error: invalid object key (must be a string)
             "type": "membership",     },     {       "file": "morph/w
                     (right here) ------^
```

(We've toyed with making this use a JSON5 parser just to avoid this entire class of problem, but haven't quite found the tuits yet)